packages/teams-js/test/public/nestedAppAuth.spec.ts

```diff
@@ -63,4 +63,97 @@ describe('nestedAppAuth', () => {
     utils.setRuntimeConfig(runtimeConfig);
     expect(nestedAppAuth.isNAAChannelRecommended()).toBeFalsy();
   });
+
+  it('should return false; isNAAChannelRecommended: false, ClientType: macos', async () => {
+    await utils.initializeWithContext(FrameContexts.content, HostClientType.macos);
+    const runtimeConfig: Runtime = {
+      apiVersion: 4,
+      supports: {},
+      isNAAChannelRecommended: false,
+    };
+    utils.setRuntimeConfig(runtimeConfig);
+    expect(nestedAppAuth.isNAAChannelRecommended()).toBeFalsy();
+  });
+
+  it('should return false; isNAAChannelRecommended: false, ClientType: desktop', async () => {
+    await utils.initializeWithContext(FrameContexts.content, HostClientType.desktop);
+    const runtimeConfig: Runtime = {
+      apiVersion: 4,
+      supports: {},
+      isNAAChannelRecommended: false,
+    };
+    utils.setRuntimeConfig(runtimeConfig);
+    expect(nestedAppAuth.isNAAChannelRecommended()).toBeFalsy();
+  });
+
+  it('should return false; isNAAChannelRecommended: false, ClientType: web', async () => {
+    await utils.initializeWithContext(FrameContexts.content, HostClientType.web);
+    const runtimeConfig: Runtime = {
+      apiVersion: 4,
+      supports: {},
+      isNAAChannelRecommended: false,
+    };
+    utils.setRuntimeConfig(runtimeConfig);
+    expect(nestedAppAuth.isNAAChannelRecommended()).toBeFalsy();
+  });
+
+  it('should return false; isNAAChannelRecommended: false, isLegacyTeams: false', async () => {
+    await utils.initializeWithContext(FrameContexts.content, HostClientType.android);
+    const runtimeConfig: Runtime = {
+      apiVersion: 4,
+      supports: {},
+      isNAAChannelRecommended: false,
+      isLegacyTeams: false,
+    };
+    utils.setRuntimeConfig(runtimeConfig);
+    expect(nestedAppAuth.isNAAChannelRecommended()).toBeFalsy();
+  });
+
+  it('should return false; isNAAChannelRecommended: false, isLegacyTeams: true, ClientType: android, does not supports nestedAppAuth', async () => {
```

Comment:
nit: Overall changes look good, but it would be great if you could change the title of all the test to a simple sentence like: should return false for the legacy Teams mobile Android client that does not support nestedAppAuth.

Reply:
addressed the comment.